pkg/operator/ceph/csi/cluster_config.go

```diff
@@ -131,31 +131,14 @@ func CreateCsiConfigMap(namespace string, clientset kubernetes.Interface) (*v1.C
 		ConfigKey: "[]",
 	}
 
-	created, err := clientset.CoreV1().ConfigMaps(namespace).Create(configMap)
+	_, err := clientset.CoreV1().ConfigMaps(namespace).Create(configMap)
 	if err != nil {
 		if !k8serrors.IsAlreadyExists(err) {
-			return nil, errors.Wrapf(err, "failed to create initial csi config map %v (in %v)", configMap.Name, namespace)
+			return errors.Wrapf(err, "failed to create initial csi config map %v (in %v)", configMap.Name, namespace)
```

Comment:
```suggestion
			return errors.Wrapf(err, "failed to create initial csi config map %q (in %q)", configMap.Name, namespace)
```

I know you merely changed the line because of the return but while we are at it...

Reply:
?